src/runtimes/index.ts

```diff
@@ -4,9 +4,11 @@ import { Config, getConfigForFunction, FunctionWithoutConfig } from '../config.j
 import { defaultFlags, FeatureFlags } from '../feature_flags.js'
 import { FunctionSource } from '../function.js'
 import type { RuntimeCache } from '../utils/cache.js'
+import { FunctionBundlingUserError } from '../utils/error.js'
 
 import goRuntime from './go/index.js'
 import jsRuntime from './node/index.js'
+import { UNIQUE_ENTRY_FILE_NAME } from './node/utils/entry_file.js'
```

Comment:
nit: Can this be called just `ENTRY_FILE_NAME`? Not sure what "unique" is adding here.